src/andromede/input_converter/src/cc_configuration/batteries.yaml

```diff
@@ -0,0 +1,119 @@
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+
+# Configuration file to convert a complex modelling artefact in an Antares study into an Andromede model.
+
+loop: 
+    
+  iteration_key: zone #Key on which we iterate (for a given list of zone, we want to execute a loop : for zone in list_zone)
+  #In the following, the notation $zone$ means the value of the string variable "zone"
+  andromede-model: antares-historic.short-term-storage #we want to create components of such a model, with id being indexed by the iteration_key
+  andromede-component-id : battery_$zone$
+
+      
+  andromede-model-parameters-to-set-a-priori:
+    #TODO : we still have to see if the following parameters can be set a priori or have to be read in the Antares study,during the conversion.
+    #- id: lower_rule_curve
```

Comment:
Does the current modelling of batteries with the binding constraints define lower/upper rule curves (maybe implicitly ?) ? Otherwise set `lower_rule_curve` to 0 and `upper-rule-curve` to 1